CheatSheets/html-cheatsheet.md

```diff
@@ -66,14 +69,30 @@
 | `<footer> ... </footer>`         | Defines a footer for a document or section                                                  |
 | `<header> ... </header>`         | Specifies a header for a document or section                                                |
 | `<main> ... </main>`             | Specifies the main content of a document                                                    |
-| `<mark> ... </mark>`             | Defines marked/highlighted text                                                             |
 | `<nav> ... </nav>`               | Defines navigation links                                                                    |
 | `<section> ... </section>`       | Defines a section in a document                                                             |
 | `<summary> ... </summary>`       | Defines a visible heading for a `<details>` element                                         |
 | `<time> ... </time>`             | Defines a date/time                                                                         |
 
 **[🔼Back to Top](#table-of-contents)**
 
+## Formatting
+
+| Command                  | Description                                          |
+| ------------------------ | ---------------------------------------------------- |
+| `<b> ... </b>`           | Defines bold text                                    |
+| `<em> ... </em>`         | Defines emphasized text                              |
+| `<i> ... </i>`           | Defines a part of text in an alternate voice or mood |
+| `<small> ... </small>`   | Defines smaller text                                 |
+| `<strong> ... </strong>` | Defines important text                               |
```

Comment:
```suggestion
```
Those tags already exist. Please `delete` the `older ones`.

Reply:
sure got it